roachtest: django failed

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21342932?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21342932?buildTab=artifacts#/django) on master @ [b5debe9e969f3b23e133510abd5c013d52c5a515](https://github.com/cockroachdb/cockroach/commits/b5debe9e969f3b23e133510abd5c013d52c5a515):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-b5debe9e969f3b23e133510abd5c013d52c5a515
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21342932-1778306184-12-n1cpu16-0001 | 20.127.200.41 | 10.1.0.107 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>metamorphicDRPC=true</code>
 - <code>metamorphicLeases=epoch</code>
 - <code>metamorphicWriteBuffering=true</code>
 - <code>runtimeAssertionsBuild=false</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21342932/django/1778306978976/1778309341009)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21342932-1778306184-12-n1cpu16&from_ts=1778306978976&to_ts=1778309341009&live=false&storage=flex_tier)

</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-63776

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21342921?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21342921?buildTab=artifacts#/django) on master @ [b5debe9e969f3b23e133510abd5c013d52c5a515](https://github.com/cockroachdb/cockroach/commits/b5debe9e969f3b23e133510abd5c013d52c5a515):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-b5debe9e969f3b23e133510abd5c013d52c5a515
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21342921-1778308286-19-n1cpu16-0001 | 35.196.214.33 | 10.142.2.214 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>metamorphicDRPC=true</code>
 - <code>metamorphicLeases=default</code>
 - <code>metamorphicWriteBuffering=true</code>
 - <code>runtimeAssertionsBuild=false</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21342921/django/1778314408722/1778317492952)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21342921-1778308286-19-n1cpu16&from_ts=1778314408722&to_ts=1778317492952&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21343291?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21343291?buildTab=artifacts#/django) on master @ [b5debe9e969f3b23e133510abd5c013d52c5a515](https://github.com/cockroachdb/cockroach/commits/b5debe9e969f3b23e133510abd5c013d52c5a515):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-b5debe9e969f3b23e133510abd5c013d52c5a515
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21343291-1778392082-18-n1cpu16-0001 | 20.127.160.185 | 10.1.0.118 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>metamorphicDRPC=true</code>
 - <code>metamorphicLeases=default</code>
 - <code>metamorphicWriteBuffering=true</code>
 - <code>runtimeAssertionsBuild=false</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21343291/django/1778392499484/1778394559426)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21343291-1778392082-18-n1cpu16&from_ts=1778392499484&to_ts=1778394559426&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21343280?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21343280?buildTab=artifacts#/django) on master @ [b5debe9e969f3b23e133510abd5c013d52c5a515](https://github.com/cockroachdb/cockroach/commits/b5debe9e969f3b23e133510abd5c013d52c5a515):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-b5debe9e969f3b23e133510abd5c013d52c5a515
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21343280-1778393841-19-n1cpu16-0001 | 34.23.192.13 | 10.142.15.231 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=true</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21343280/django/1778395373780/1778398931186)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21343280-1778393841-19-n1cpu16&from_ts=1778395373780&to_ts=1778398931186&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21343597?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21343597?buildTab=artifacts#/django) on master @ [1fcef18003b8dec750ed636aa92d35ca10105878](https://github.com/cockroachdb/cockroach/commits/1fcef18003b8dec750ed636aa92d35ca10105878):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-1fcef18003b8dec750ed636aa92d35ca10105878
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21343597-1778479091-14-n1cpu16-0001 | 74.235.160.77 | 10.1.0.100 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>metamorphicLeases=expiration</code>
 - <code>metamorphicWriteBuffering=true</code>
 - <code>runtimeAssertionsBuild=false</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21343597/django/1778479500871/1778481661549)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21343597-1778479091-14-n1cpu16&from_ts=1778479500871&to_ts=1778481661549&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21343586?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21343586?buildTab=artifacts#/django) on master @ [1fcef18003b8dec750ed636aa92d35ca10105878](https://github.com/cockroachdb/cockroach/commits/1fcef18003b8dec750ed636aa92d35ca10105878):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-1fcef18003b8dec750ed636aa92d35ca10105878
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21343586-1778481151-13-n1cpu16-0001 | 34.24.155.100 | 10.142.0.255 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>metamorphicDRPC=true</code>
 - <code>metamorphicLeases=expiration</code>
 - <code>runtimeAssertionsBuild=false</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21343586/django/1778482301132/1778485526946)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21343586-1778481151-13-n1cpu16&from_ts=1778482301132&to_ts=1778485526946&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21344536?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21344536?buildTab=artifacts#/django) on master @ [6cf0a87806c012a2eb918fd9a58d5549d40ef39d](https://github.com/cockroachdb/cockroach/commits/6cf0a87806c012a2eb918fd9a58d5549d40ef39d):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-6cf0a87806c012a2eb918fd9a58d5549d40ef39d
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21344536-1778565590-13-n1cpu16-0001 | 20.62.194.102 | 10.1.0.103 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>metamorphicDRPC=true</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=true</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21344536/django/1778566455590/1778568937556)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21344536-1778565590-13-n1cpu16&from_ts=1778566455590&to_ts=1778568937556&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21344525?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21344525?buildTab=artifacts#/django) on master @ [6cf0a87806c012a2eb918fd9a58d5549d40ef39d](https://github.com/cockroachdb/cockroach/commits/6cf0a87806c012a2eb918fd9a58d5549d40ef39d):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-6cf0a87806c012a2eb918fd9a58d5549d40ef39d
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/cpu_arch=arm64/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21344525-1778567570-17-n1cpu16-0001 | 136.116.10.107 | 10.128.0.8 |

Parameters:
 - <code>arch=arm64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>metamorphicLeases=default</code>
 - <code>metamorphicWriteBuffering=true</code>
 - <code>runtimeAssertionsBuild=false</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21344525/django/1778569994625/1778573203189)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21344525-1778567570-17-n1cpu16&from_ts=1778569994625&to_ts=1778573203189&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21345724?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21345724?buildTab=artifacts#/django) on master @ [39d3ad55242ae3760945906de6b828aabb3439f2](https://github.com/cockroachdb/cockroach/commits/39d3ad55242ae3760945906de6b828aabb3439f2):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-39d3ad55242ae3760945906de6b828aabb3439f2
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21345724-1778651873-17-n1cpu16-0001 | 172.206.227.114 | 10.1.0.130 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>metamorphicDRPC=true</code>
 - <code>metamorphicLeases=epoch</code>
 - <code>runtimeAssertionsBuild=false</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21345724/django/1778652783968/1778655195005)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21345724-1778651873-17-n1cpu16&from_ts=1778652783968&to_ts=1778655195005&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21345713?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21345713?buildTab=artifacts#/django) on master @ [39d3ad55242ae3760945906de6b828aabb3439f2](https://github.com/cockroachdb/cockroach/commits/39d3ad55242ae3760945906de6b828aabb3439f2):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-39d3ad55242ae3760945906de6b828aabb3439f2
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21345713-1778654091-14-n1cpu16-0001 | 34.24.56.239 | 10.142.2.193 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=true</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21345713/django/1778655259102/1778658868052)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21345713-1778654091-14-n1cpu16&from_ts=1778655259102&to_ts=1778658868052&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21348681?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21348681?buildTab=artifacts#/django) on master @ [b0930f9cb4c4da2eebd23056fc2ee6792e20c6da](https://github.com/cockroachdb/cockroach/commits/b0930f9cb4c4da2eebd23056fc2ee6792e20c6da):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-b0930f9cb4c4da2eebd23056fc2ee6792e20c6da
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21348681-1778824795-08-n1cpu16-0001 | 52.146.88.110 | 10.1.0.10 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>metamorphicDRPC=true</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=true</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21348681/django/1778825148147/1778827568317)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21348681-1778824795-08-n1cpu16&from_ts=1778825148147&to_ts=1778827568317&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21348670?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21348670?buildTab=artifacts#/django) on master @ [b0930f9cb4c4da2eebd23056fc2ee6792e20c6da](https://github.com/cockroachdb/cockroach/commits/b0930f9cb4c4da2eebd23056fc2ee6792e20c6da):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-b0930f9cb4c4da2eebd23056fc2ee6792e20c6da
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/cpu_arch=arm64/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21348670-1778826635-88-n1cpu16-0001 | 34.42.160.161 | 10.128.0.79 |

Parameters:
 - <code>arch=arm64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=false</code>
 - <code>metamorphicLeases=epoch</code>
 - <code>metamorphicWriteBuffering=true</code>
 - <code>runtimeAssertionsBuild=false</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21348670/django/1778841601534/1778844788123)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21348670-1778826635-88-n1cpu16&from_ts=1778841601534&to_ts=1778844788123&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21349939?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21349939?buildTab=artifacts#/django) on master @ [079d681ef220f807f276a9fef1dee92ffa552a2a](https://github.com/cockroachdb/cockroach/commits/079d681ef220f807f276a9fef1dee92ffa552a2a):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-079d681ef220f807f276a9fef1dee92ffa552a2a
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21349939-1778911000-20-n1cpu16-0001 | 20.106.194.19 | 10.1.0.132 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>metamorphicLeases=expiration</code>
 - <code>metamorphicWriteBuffering=true</code>
 - <code>runtimeAssertionsBuild=false</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21349939/django/1778911393348/1778913235792)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21349939-1778911000-20-n1cpu16&from_ts=1778911393348&to_ts=1778913235792&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21349927?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21349927?buildTab=artifacts#/django) on master @ [079d681ef220f807f276a9fef1dee92ffa552a2a](https://github.com/cockroachdb/cockroach/commits/079d681ef220f807f276a9fef1dee92ffa552a2a):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-079d681ef220f807f276a9fef1dee92ffa552a2a
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21349927-1778913136-67-n1cpu16-0001 | 34.26.93.55 | 10.142.3.180 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicDRPC=true</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=false</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21349927/django/1778928386838/1778931352154)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21349927-1778913136-67-n1cpu16&from_ts=1778928386838&to_ts=1778931352154&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21350417?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21350417?buildTab=artifacts#/django) on master @ [b73593d0150a855f82ee398076748436dabee403](https://github.com/cockroachdb/cockroach/commits/b73593d0150a855f82ee398076748436dabee403):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-b73593d0150a855f82ee398076748436dabee403
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21350417-1778997486-21-n1cpu16-0001 | 172.203.241.226 | 10.1.0.149 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=true</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21350417/django/1778997972468/1779000434092)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21350417-1778997486-21-n1cpu16&from_ts=1778997972468&to_ts=1779000434092&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21350406?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21350406?buildTab=artifacts#/django) on master @ [b73593d0150a855f82ee398076748436dabee403](https://github.com/cockroachdb/cockroach/commits/b73593d0150a855f82ee398076748436dabee403):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-b73593d0150a855f82ee398076748436dabee403
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21350406-1778999587-63-n1cpu16-0001 | 34.148.232.5 | 10.142.1.81 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=true</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21350406/django/1779015390760/1779019075709)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21350406-1778999587-63-n1cpu16&from_ts=1779015390760&to_ts=1779019075709&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21350868?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21350868?buildTab=artifacts#/django) on master @ [b73593d0150a855f82ee398076748436dabee403](https://github.com/cockroachdb/cockroach/commits/b73593d0150a855f82ee398076748436dabee403):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-b73593d0150a855f82ee398076748436dabee403
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21350868-1779084057-19-n1cpu16-0001 | 20.55.84.93 | 10.1.0.126 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>metamorphicDRPC=true</code>
 - <code>metamorphicLeases=epoch</code>
 - <code>runtimeAssertionsBuild=true</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21350868/django/1779084430059/1779086802937)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21350868-1779084057-19-n1cpu16&from_ts=1779084430059&to_ts=1779086802937&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21352085?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21352085?buildTab=artifacts#/django) on master @ [322d13b08e95f4c342cb6e8418cf439a943242f3](https://github.com/cockroachdb/cockroach/commits/322d13b08e95f4c342cb6e8418cf439a943242f3):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-322d13b08e95f4c342cb6e8418cf439a943242f3
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21352085-1779170183-19-n1cpu16-0001 | 40.76.101.39 | 10.1.0.127 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>metamorphicDRPC=true</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=true</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21352085/django/1779170565687/1779172933582)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21352085-1779170183-19-n1cpu16&from_ts=1779170565687&to_ts=1779172933582&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21353385?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21353385?buildTab=artifacts#/django) on master @ [d3d7be79ccdc6ee14ea4f75bcb241f326ed3e160](https://github.com/cockroachdb/cockroach/commits/d3d7be79ccdc6ee14ea4f75bcb241f326ed3e160):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-d3d7be79ccdc6ee14ea4f75bcb241f326ed3e160
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21353385-1779256055-09-n1cpu16-0001 | 20.55.88.103 | 10.1.0.10 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=true</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21353385/django/1779256417211/1779258839180)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21353385-1779256055-09-n1cpu16&from_ts=1779256417211&to_ts=1779258839180&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21355030?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21355030?buildTab=artifacts#/django) on master @ [f4de542b4e2d0a3074d9590a220500c1dd211caa](https://github.com/cockroachdb/cockroach/commits/f4de542b4e2d0a3074d9590a220500c1dd211caa):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-f4de542b4e2d0a3074d9590a220500c1dd211caa
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21355030-1779343095-21-n1cpu16-0001 | 20.55.81.248 | 10.1.0.140 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>metamorphicLeases=expiration</code>
 - <code>metamorphicWriteBuffering=true</code>
 - <code>runtimeAssertionsBuild=false</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21355030/django/1779343507391/1779345437906)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21355030-1779343095-21-n1cpu16&from_ts=1779343507391&to_ts=1779345437906&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21355019?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21355019?buildTab=artifacts#/django) on master @ [f4de542b4e2d0a3074d9590a220500c1dd211caa](https://github.com/cockroachdb/cockroach/commits/f4de542b4e2d0a3074d9590a220500c1dd211caa):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-f4de542b4e2d0a3074d9590a220500c1dd211caa
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21355019-1779345071-70-n1cpu16-0001 | 35.190.160.192 | 10.142.0.92 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicDRPC=true</code>
 - <code>metamorphicLeases=epoch</code>
 - <code>runtimeAssertionsBuild=false</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21355019/django/1779358290487/1779361257872)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21355019-1779345071-70-n1cpu16&from_ts=1779358290487&to_ts=1779361257872&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21356543?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21356543?buildTab=artifacts#/django) on master @ [248e519687c1d8129f5675a0fe0db9cf125c2b6d](https://github.com/cockroachdb/cockroach/commits/248e519687c1d8129f5675a0fe0db9cf125c2b6d):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-248e519687c1d8129f5675a0fe0db9cf125c2b6d
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21356543-1779428963-06-n1cpu16-0001 | 20.102.108.183 | 10.1.0.129 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>metamorphicDRPC=true</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=false</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21356543/django/1779429356942/1779431288681)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21356543-1779428963-06-n1cpu16&from_ts=1779429356942&to_ts=1779431288681&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21356532?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21356532?buildTab=artifacts#/django) on master @ [248e519687c1d8129f5675a0fe0db9cf125c2b6d](https://github.com/cockroachdb/cockroach/commits/248e519687c1d8129f5675a0fe0db9cf125c2b6d):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-248e519687c1d8129f5675a0fe0db9cf125c2b6d
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21356532-1779430557-113-n1cpu16-0001 | 34.138.194.252 | 10.142.1.157 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicDRPC=true</code>
 - <code>metamorphicLeases=expiration</code>
 - <code>metamorphicWriteBuffering=true</code>
 - <code>runtimeAssertionsBuild=true</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21356532/django/1779452468734/1779455906629)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21356532-1779430557-113-n1cpu16&from_ts=1779452468734&to_ts=1779455906629&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21357926?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21357926?buildTab=artifacts#/django) on master @ [971ba39ae57f213d10f07b1cadad5404361fa66f](https://github.com/cockroachdb/cockroach/commits/971ba39ae57f213d10f07b1cadad5404361fa66f):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-971ba39ae57f213d10f07b1cadad5404361fa66f
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21357926-1779515792-14-n1cpu16-0001 | 74.235.185.243 | 10.1.0.123 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>metamorphicDRPC=true</code>
 - <code>metamorphicLeases=expiration</code>
 - <code>runtimeAssertionsBuild=false</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21357926/django/1779516207521/1779518229067)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21357926-1779515792-14-n1cpu16&from_ts=1779516207521&to_ts=1779518229067&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21357915?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21357915?buildTab=artifacts#/django) on master @ [971ba39ae57f213d10f07b1cadad5404361fa66f](https://github.com/cockroachdb/cockroach/commits/971ba39ae57f213d10f07b1cadad5404361fa66f):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-971ba39ae57f213d10f07b1cadad5404361fa66f
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21357915-1779518049-17-n1cpu16-0001 | 35.237.16.19 | 10.142.2.163 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>metamorphicLeases=default</code>
 - <code>metamorphicWriteBuffering=true</code>
 - <code>runtimeAssertionsBuild=true</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21357915/django/1779519222889/1779522596386)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21357915-1779518049-17-n1cpu16&from_ts=1779519222889&to_ts=1779522596386&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21358361?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21358361?buildTab=artifacts#/django) on master @ [4e7fa9411f645a8bf0f8baf0bb1147118d0cf730](https://github.com/cockroachdb/cockroach/commits/4e7fa9411f645a8bf0f8baf0bb1147118d0cf730):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-4e7fa9411f645a8bf0f8baf0bb1147118d0cf730
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21358361-1779602273-48-n1cpu16-0001 | 20.106.196.63 | 10.1.0.167 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>fs=ext4</code>
 - <code>localSSD=true</code>
 - <code>metamorphicDRPC=true</code>
 - <code>metamorphicLeases=expiration</code>
 - <code>metamorphicWriteBuffering=true</code>
 - <code>runtimeAssertionsBuild=false</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21358361/django/1779606645849/1779608577554)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21358361-1779602273-48-n1cpu16&from_ts=1779606645849&to_ts=1779608577554&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21358349?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21358349?buildTab=artifacts#/django) on master @ [4e7fa9411f645a8bf0f8baf0bb1147118d0cf730](https://github.com/cockroachdb/cockroach/commits/4e7fa9411f645a8bf0f8baf0bb1147118d0cf730):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-4e7fa9411f645a8bf0f8baf0bb1147118d0cf730
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21358349-1779604344-15-n1cpu16-0001 | 104.196.113.200 | 10.142.3.42 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>metamorphicDRPC=true</code>
 - <code>metamorphicLeases=leader</code>
 - <code>metamorphicWriteBuffering=true</code>
 - <code>runtimeAssertionsBuild=true</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21358349/django/1779605468108/1779608883180)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21358349-1779604344-15-n1cpu16&from_ts=1779605468108&to_ts=1779608883180&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21358711?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21358711?buildTab=artifacts#/django) on master @ [b852382ca5d1b265a84370597f0bd77c0ab110cb](https://github.com/cockroachdb/cockroach/commits/b852382ca5d1b265a84370597f0bd77c0ab110cb):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-b852382ca5d1b265a84370597f0bd77c0ab110cb
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21358711-1779688740-17-n1cpu16-0001 | 20.127.184.5 | 10.1.0.111 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=false</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21358711/django/1779689210000/1779691113211)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21358711-1779688740-17-n1cpu16&from_ts=1779689210000&to_ts=1779691113211&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21358700?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21358700?buildTab=artifacts#/django) on master @ [b852382ca5d1b265a84370597f0bd77c0ab110cb](https://github.com/cockroachdb/cockroach/commits/b852382ca5d1b265a84370597f0bd77c0ab110cb):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-b852382ca5d1b265a84370597f0bd77c0ab110cb
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21358700-1779690798-15-n1cpu16-0001 | 35.237.224.151 | 10.142.3.194 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>metamorphicLeases=expiration</code>
 - <code>runtimeAssertionsBuild=true</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21358700/django/1779691846488/1779695384594)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21358700-1779690798-15-n1cpu16&from_ts=1779691846488&to_ts=1779695384594&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21359445?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21359445?buildTab=artifacts#/django) on master @ [8cf7d10d20eac900a3cda4306090d7b02ae716e7](https://github.com/cockroachdb/cockroach/commits/8cf7d10d20eac900a3cda4306090d7b02ae716e7):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-8cf7d10d20eac900a3cda4306090d7b02ae716e7
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21359445-1779775048-18-n1cpu16-0001 | 74.235.160.192 | 10.1.0.116 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>metamorphicLeases=expiration</code>
 - <code>runtimeAssertionsBuild=true</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21359445/django/1779775492010/1779777991891)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21359445-1779775048-18-n1cpu16&from_ts=1779775492010&to_ts=1779777991891&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21359433?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21359433?buildTab=artifacts#/django) on master @ [8cf7d10d20eac900a3cda4306090d7b02ae716e7](https://github.com/cockroachdb/cockroach/commits/8cf7d10d20eac900a3cda4306090d7b02ae716e7):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-8cf7d10d20eac900a3cda4306090d7b02ae716e7
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21359433-1779776956-15-n1cpu16-0001 | 35.231.38.169 | 10.142.3.128 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>metamorphicLeases=leader</code>
 - <code>runtimeAssertionsBuild=false</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21359433/django/1779778030722/1779781011078)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21359433-1779776956-15-n1cpu16&from_ts=1779778030722&to_ts=1779781011078&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21361139?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyAzureBazel/21361139?buildTab=artifacts#/django) on master @ [a081a00a50d84e8da201bbe55b898ec41b9b013b](https://github.com/cockroachdb/cockroach/commits/a081a00a50d84e8da201bbe55b898ec41b9b013b):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-a081a00a50d84e8da201bbe55b898ec41b9b013b
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21361139-1779861522-26-n1cpu16-0001 | 20.127.173.77 | 10.1.0.151 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=azure</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>metamorphicLeases=default</code>
 - <code>runtimeAssertionsBuild=true</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21361139/django/1779862142159/1779864584653)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21361139-1779861522-26-n1cpu16&from_ts=1779862142159&to_ts=1779864584653&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21361128?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/21361128?buildTab=artifacts#/django) on master @ [a081a00a50d84e8da201bbe55b898ec41b9b013b](https://github.com/cockroachdb/cockroach/commits/a081a00a50d84e8da201bbe55b898ec41b9b013b):

Failed with:

```
(orm_helpers.go:225).summarizeFailed: 
Tests run on Cockroach v26.3.0-alpha.00000000-dev-a081a00a50d84e8da201bbe55b898ec41b9b013b
Tests run against django cockroach-4.1.x
9957 Total Tests Run
9953 tests passed
4 tests failed
634 tests skipped
1 test ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
--- FAIL: introspection.tests.IntrospectionTests.test_get_constraints (unexpected)
For a full summary look at the django artifacts 
An updated blocklist (djangoBlocklist) is available in the artifacts' django log
test artifacts and logs in: /artifacts/django/run_1
```
Cluster Node to Ip Mapping:
| Node | Public IP | Private IP |
| --- | --- | --- |
| teamcity-21361128-1779863753-14-n1cpu16-0001 | 34.138.32.161 | 10.142.3.171 |

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=16</code>
 - <code>diskCount=0</code>
 - <code>encrypted=false</code>
 - <code>metamorphicLeases=default</code>
 - <code>metamorphicWriteBuffering=true</code>
 - <code>runtimeAssertionsBuild=false</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-21361128/django/1779864863239/1779867735798)



See: [Datadog Logs](https://us5.datadoghq.com/logs?query=service%3Aroachtest+%40cluster%3Ateamcity-21361128-1779863753-14-n1cpu16&from_ts=1779864863239&to_ts=1779867735798&live=false&storage=flex_tier)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/cmd/roachtest/tests/django.go
+++ b/pkg/cmd/roachtest/tests/django.go
@@ -24,8 +24,8 @@ var djangoCockroachDBReleaseTagRegex = regexp.MustCompile(`^(?P<major>\d+)\.(?P<
 
 // WARNING: DO NOT MODIFY the name of the below constant/variable without approval from the docs team.
 // This is used by docs automation to produce a list of supported versions for ORM's.
-var djangoSupportedTag = "cockroach-4.1.x"
-var djangoCockroachDBSupportedTag = "4.1.*"
+var djangoSupportedTag = "cockroach-6.0.x"
+var djangoCockroachDBSupportedTag = "6.0.*"
 
 func registerDjango(r registry.Registry) {
 	runDjango := func(
@@ -65,22 +65,22 @@ func registerDjango(r registry.Registry) {
 			c,
 			node,
 			"install dependencies",
-			`sudo apt-get -qq install make python3.10 libpq-dev python3.10-dev gcc python3-virtualenv python3-setuptools python-setuptools build-essential python3.10-distutils python3-apt libmemcached-dev`,
+			`sudo apt-get -qq install make python3.12 libpq-dev python3.12-dev gcc python3-virtualenv python3-setuptools python-setuptools build-essential python3.12-distutils python3-apt libmemcached-dev`,
 		); err != nil {
 			t.Fatal(err)
 		}
 
 		if err := repeatRunE(
-			ctx, t, c, node, "set python3.10 as default", `
-    		sudo update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.10 1
-    		sudo update-alternatives --config python3`,
+			ctx, t, c, node, "set python3.12 as default", `
+			sudo update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.12 1
+			sudo update-alternatives --config python3`,
 		); err != nil {
 			t.Fatal(err)
 		}
 
 		if err := repeatRunE(
 			ctx, t, c, node, "install pip",
-			`curl https://bootstrap.pypa.io/get-pip.py | sudo -H python3.10`,
+			`curl https://bootstrap.pypa.io/get-pip.py | sudo -H python3.12`,
 		); err != nil {
 			t.Fatal(err)
 		}
@@ -263,7 +263,6 @@ PASSWORD_HASHERS = [
     'django.contrib.auth.hashers.MD5PasswordHasher',
 ]
 TEST_RUNNER = '.cockroach_settings.NonDescribingDiscoverRunner'
-DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'
 
 class NonDescribingDiscoverRunner(DiscoverRunner):
     def get_test_runner_kwargs(self):

--- a/pkg/cmd/roachtest/tests/django_blocklist.go
+++ b/pkg/cmd/roachtest/tests/django_blocklist.go
@@ -157,11 +157,7 @@ var enabledDjangoTests = []string{
 }
 
 // Maintain that this list is alphabetized.
-var djangoBlocklist = blocklist{
-	`schema.tests.SchemaTests.test_alter_text_field_to_date_field`:     "alter type requires USING",
-	`schema.tests.SchemaTests.test_alter_text_field_to_datetime_field`: "alter type requires USING",
-	`schema.tests.SchemaTests.test_alter_text_field_to_time_field`:     "alter type requires USING",
-}
+var djangoBlocklist = blocklist{}
 
 var djangoIgnoreList = blocklist{
 	`select_for_update.tests.SelectForUpdateTests.test_nowait_raises_error_on_block`: "flaky; see #120196",

